test/e2e/multiparty/fabric_contracts.go

```diff
@@ -163,8 +163,8 @@ func (suite *FabricContractTestSuite) TestE2EContractEvents() {
 	assert.NoError(suite.T(), err)
 	assert.Equal(suite.T(), assetName, res.(map[string]interface{})["name"])
 
-	suite.testState.client1.DeleteContractListener(suite.T(), subs[0].ID)
-	subs = suite.testState.client1.GetContractListeners(suite.T(), suite.testState.startTime)
-	assert.Equal(suite.T(), 0, len(subs))
```

Comment:
Did you mean to leave this commented out?

Reply:
No 🤦